src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Color color = colorSupplier.getRandomColor();
```

Comment:
Now you create random color only once when application starts. This is not correct. All your figures will have same color.
You should generate new random color each time you create random figure
```suggestion
```